packages/um7/package.py

```diff
@@ -13,6 +13,21 @@
 import os
 import os.path
 
+def _bld_cfg_path(opt_value, hg):
+    """
+    Return the path to the build configuration, depending on 
+    "opt" value (opt_value) and HadGEM number (hg).
+    """
+    if opt_value == "debug":
+        bld_config = f"bld-dbg-hadgem{hg}-C2.cfg"
+    else:
+        bld_config = f"bld-hadgem{hg}-mct.cfg"
+    return join_path(f"ummodel_hg{hg}", "cfg", bld_config)
+
+
+_hg = 3  # build HadGEM3 ONLY here
```

Comment:
Make it a member variable of the class `Um7`.

Reply:
I will just hardcode the 3 since it will never change.

Reply:
Done.